### PR TITLE
Added download link for PointNav baseline weights to ReadMe.

### PR DESCRIPTION
**Pull request by Skylion007**
* Also added missing pip install to Docker and set the number of test episodes explicitly for compatibility with older habitat-api versions.

Added download link for our DD-PPO-PointNav baseline model.

### CODE REVIEW

File: configs/ddppo_pointnav.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ TORCH_GPU_ID: 0
 VIDEO_OPTION: []
 TENSORBOARD_DIR: "tb"
 VIDEO_DIR: "video_dir"
-TEST_EPISODE_COUNT: -1
+TEST_EPISODE_COUNT: 994
```
Comment (dhruvbatra) [inline configs/ddppo_pointnav.yaml:9]: Is this change intentional?

Comment (Skylion007) [reply configs/ddppo_pointnav.yaml:9]: Yeah, because I was worried it might cause issues with people still on habitat-stable. Should I change it back?

Comment (dhruvbatra) [reply configs/ddppo_pointnav.yaml:9]: @erikwijmans -- what do you suggest?

Comment (erikwijmans) [reply configs/ddppo_pointnav.yaml:9]: Habitat stable isn't useable for the challenge as is, so I'm not worried about that.  I think the challenge 2020 tag is usable however (which is before the PR needed for -1).  994 is fine as it should be clear that this config is the pointnav dataset.  The reason to do -1 instead of 994 is if we think people will use this config for other datasets and not understand that they need to change the 994.
